.codeclimate.yml

```
@@ -0,0 +1,4 @@
+## Exclude auto generated files
+exclude_patterns:
+  - "internal/"
```

Maybe `internal/api/` -- in case there are other (non-autogenerated) files in `internal/` in the future?

```suggestion
  - "internal/api/"
```